fix buck file issue of XNNPACK

digantdesai opened the pull request:
Summary: trying to fix https://fb.workplace.com/groups/fbcode/posts/6015466968490092/?comment_id=6015968131773309&reply_comment_id=6018180184885437&notif_id=1678735231098965&notif_t=work_group_comment_reply

Test Plan: buck2 test

Differential Revision: D44034833

kit1980 commented:
Consider doing something like this https://github.com/pytorch/pytorch/pull/94935

tiandiao123 commented:
> 

oh, thank you! let me try it.

Commit:
fix buck file issue of XNNPACK (#96689)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/96689

trying to fix https://fb.workplace.com/groups/fbcode/posts/6015466968490092/?comment_id=6015968131773309&reply_comment_id=6018180184885437&notif_id=1678735231098965&notif_t=work_group_comment_reply

Test Plan: buck2 test fbcode//caffe2/test:xnnpack_integration

Reviewed By: dmm-fb

Differential Revision: D44034833

fbshipit-source-id: 6e064bada08f3c8b52aa2bc18d25df2d8cb4abf7

tiandiao123 commented:
it is not required to change.
